canopen_monitor/parse/eds.py

```diff
@@ -317,7 +321,7 @@ def __init__(self, eds_data: [str]):
                 prev = i + 1
 
         if self.device_commissioning is not None:
-            self.node_id = convert_value(self.device_commissioning.node_id)
+            self.node_id = convert_value(self.device_commissioning.node_n_id)
```

Comment:
This appears to be a problem rooted at our free-wheeled config parsing.

&nbsp;

I played around with [configparser](https://docs.python.org/3/library/configparser.html) last night and I think since some of the bugs in #86  are related to how the parsing happens, it makes sense to try and do the switch over to the standard library in this PR.

It'll be a good deal of syntax updating, but I think it's worth the trouble.

Thoughts?

Reply:
After taking a look at the documentation, it seems like it would be a worthwhile investment of time. Definitely makes navigating the DCF in code a lot simpler. I can do some testing on my side to build a little bit of familiarity.